pkgs/tools/backup/awsbck/default.nix

```
@@ -0,0 +1,34 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, rustPlatform
+, installShellFiles
+, Security
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "awsbck";
+  version = "0.3.2";
+
+  src = fetchFromGitHub {
+    owner = "beeb";
+    repo = pname;
```

Don't use `pname` inside `fetchFromGitHub`.

```suggestion
    repo = "awsbck";
```

---

Not sure why as they are equivalent, but done anyway! Explanation welcome :)

---

I don't know the exact reason (I actually forgo). Apparently this is a trend in `nixpkgs`, if you look at already merged PRs, you'll see that other maintainers are asking contributors to do so.